### ISSUE TITLE
Project Status / Maintenance

### ISSUE THREAD

**Issue by 16amattice** [hoodiehq/hoodie#938]
Good Afternoon,

I read in another issue that this project is no longer being actively maintained? I was wondering who I would have to get into contact with about maintaining/continuing this project, as I think it has great potential and could turn into something great!
Thanks

**Comment by kragil** [comment]
They all ran and hid. I guess you have to fork and rename.

**Comment by zorenkonte** [comment]
Upon visiting their website [hood.ie](http://hood.ie/), you'll notice that it redirects you to https://rxdb.info/. This redirection possibly implies their endorsement of using the latter as an alternative.

**Comment by AudunVN** [comment]
> Upon visiting their website [hood.ie](http://hood.ie/), you'll notice that it redirects you to https://rxdb.info/. This redirection possibly implies their endorsement of using the latter as an alternative.

Would just like to note that this does not appear to be an endorsement, as per https://github.com/pubkey/rxdb/issues/4433 and more specifically https://github.com/pubkey/rxdb/issues/4433#issuecomment-1430117688:

> Hi @janl I was not lucky enough to grab the domain :) Instead some guy from bulgaria grabbed it ( this guy: https://seo.domains/ ) Since it was parked for so long, I wanted to buy it for the SEO-juice and the backlinks so I debated a price with him. At the moment I still not "own" the domain, I signed a rent-to-lease via https://dan.com/ where I pay monthly (348€/month) to rent it and I will get ownership of the domain if I keep paying for the next 2 years. The guy wanted 7500$ for a full buy.
> 
> I know the domain belongs to you and I have no problem with giving the domain back to you. But you will have to to discuss a price with the current owner and then we can find out how I can transfer the rent-to-lease over to you.
> 
> By the way you can also write privately with me via discord if you want to.

_Originally posted by @pubkey in https://github.com/pubkey/rxdb/issues/4433#issuecomment-1430117688_

**Comment by pubkey** [comment]
Yes @AudunVN is right. I (=RxDB) am not affiliated with hood.ie and there is no endorsement or anything. I just grabbed the domain in my growth hacking attempts for RxDB. 
As stated before I just rent the domain to get the traffic and SEO power. If hood.ie every decides to want the domain back, I will gracefully let them rent it from the current owner instead.

**Comment by 16amattice** [comment]
Well it doesn't seem like any of the original contributors want to reach out, so I may fork and rename, and start fixing some of the bigger issues...

**Comment by 16amattice** [comment]
@AudunVN Not sure if you were interested at all, but I thought I'd link to my forked repo. I am going to stop chatting here, and start replicating everything under a new name, and start replicating issues, fixing links, etc.

https://github.com/BeaconHQ/Beacon
